domain/src/test/java/org/oppia/android/domain/profile/ProfileManagementControllerTest.kt

```diff
@@ -729,6 +729,33 @@ class ProfileManagementControllerTest {
     assertThat(wasProfileEverAdded).isTrue()
   }
 
+  @Test
+  fun testAddAdminProfile_addAnotherAdminProfile_checkSecondAdminProfileWasAdded() {
```

Comment:
Rename this to `testAddAdminProfile_addAnotherAdminProfile_checkSecondAdminProfileWasNotAdded `

You missed **Not** in the sentence.

Reply:
> Really nice work. One nit suggested.

Thankyou @rt4914 
nit changes done!!